webapp/src/features/intersections/reports/report-list-table.tsx

```diff
@@ -1,56 +1,18 @@
 import React from 'react'
 import { format } from 'date-fns'
-import {
-  Avatar,
-  Box,
-  Button,
-  IconButton,
-  Table,
-  TableBody,
-  TableCell,
-  TablePagination,
-  TableRow,
-  Typography,
-} from '@mui/material'
+import { Box, Button, Table, TableBody, TableCell, TablePagination, TableRow, Typography } from '@mui/material'
```

Comment:
Is this single line/multiline formatting something we have specified and can keep consistent?

Reply:
Yes, the line length is set in the .vscode/.prettierrc.json. This only changed because I removed a few unused imports, and it automatically pulled it back into one line because it was shortened enough